Design.md

```diff
@@ -0,0 +1,238 @@
+---
+authors: Brian Green (bgreen007@cox.net)
+state: draft
+---
+
+## What
+Job worker service that provides an API to run arbitrary Linux processes.
+
+## Details
+
+
+### Security
+Use mTLS for gRPC communication. TLS version 1.3 only.
```

Comment:
Can you provide a brief justification for selecting 1.3 - I don't disagree but it'd be nice to understand your thought process.

Reply:
added a few notes regarding 1.3: faster, simpler.